toolkit/tools/imagecustomizerapi/verity.go

```diff
@@ -5,25 +5,55 @@ package imagecustomizerapi
 
 import (
 	"fmt"
+	"regexp"
+)
+
+const (
+	VerityRootDevicePath = "/dev/mapper/root"
+	VerityRootDeviceName = "root"
```

Comment:
I feel like this might be duplicated with `rootfsVerity.Name`, but if without this constant `VerityRootDeviceName`, we need to force the `rootfsVerity.Name` to be `root` when the rootfs Verity.

Reply:
I've removed `VerityRootDevicePath`.

`VerityRootDeviceName` has to stick around since we need to use it in API validation.